tests/functional/tests/__init__.py

```diff
@@ -0,0 +1 @@
+"""Package containing functional tests."""
```

Comment:
not-blocking: Why this file is needed?

Reply:
I realized that it's commonly used, because zaza is importing  tests e.g. `tests.test_charm.SoftwareInventoryCollectorTests`. Thanks marking as resolved